cluster/saltbase/salt/generate-cert/make-ca-cert.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [ "${DEBUG}" == "true" ]; then
+if [ "${DEBUG:-}" == "true" ]; then
```

Comment:
As @luxas suggest I'll wait for #21357 to be merged and revert change to this line.